spec/ordinary-and-exotic-object-behaviours.html

```diff
@@ -0,0 +1,97 @@
+          1. If _Desc_.[[Enumerable]] is present and has value *false*, return *false*.
+          1. If _Desc_.[[Configurable]] is present and has value *true*, return *false*.
+          1. If _Desc_.[[Value]] is present, return SameValue(_Desc_.[[Value]], _current_.[[Value]]).
+          1. Return *false*.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-record-exotic-objects-ownpropertykeys">
+        <h1>[[OwnPropertyKeys]] ( )</h1>
+        <p>When the [[OwnPropertyKeys]] internal method of a Record exotic object _R_ is called, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _keys_ be a new empty List.
+          1. Let _rec_ be _R_.[[RecordData]].
+          1. Assert: Type(_rec_) is Record.
+          1. Let _len_ be the length of _rec_.
+          1. For each pair _p_ of _rec_.[[Fields]], do
+            1. Let _key_ be _p_.[[Key]]
+            1. Assert: Type(_key_) is String
+            1. Add _key_ to _keys_
+          1. Return _keys_
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-recordcreate" aoid="RecordCreate">
+        <h1>RecordCreate ( _value_ )</h1>
+        <p>This abstract operation RecordCreate takes a _value_ argument. It is used to specify the creation of new Record exotic objects. It performs the following steps when called:</p>
+        <emu-alg>
+            1. Assert: Type(_value_) is Record.
+            1. Let _R_ be ! MakeBasicObject(&laquo; [[RecordData]] &raquo;).
+            1. Set _R_.[[RecordData]] to _value_
+            1. Set _R_.[[GetOwnProperty]] as specified in <emu-xref href="#sec-record-exotic-objects-getownproperty-p"></emu-xref>.
+            1. Set _R_.[[DefineOwnProperty]] as specified in <emu-xref href="#sec-record-exotic-objects-defineownproperty-p-desc"></emu-xref>.
+            1. Set _R_.[[OwnPropertyKeys]] as specified in <emu-xref href="#sec-record-exotic-objects-ownpropertykeys"></emu-xref>.
+            1. Return _R_
+            
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-recordgetownproperty-r-p" aoid="RecordGetOwnProperty">
+        <h1>RecordGetOwnProperty ( _R_, _P_ )</h1>
+        <p>This abstract operation RecordGetOwnProperty takes arguments _R_ and _P_. It performs the following steps when called:</p>
+        <emu-alg>
+            1. Assert: _R_ is an Object that has [[RecordData]] internal slot.
+            1. Assert: Type(_P_) is String.
+            1. Let _rec_ be _R_.[[RecordData]]
+            1. Assert Type(rec) is Record
+            1. Let _len_ be the length of _rec_.
+            1. Let _resultPrim_ be ~empty~
+            1. For each pair _p_ of _rec_.[[Fields]], do
+              1. Let _key_ be _p_.[[Key]]
+              1. Let _prim_ be _p_.[[Value]]
+              1. Assert: Type(_key_) is String
+              1. Assert: Type(_prim_) is not Object
+              1. If _key_ == _P_, let _resultPrim_ be _prim_
```

Comment:
Avoid ==; spec text tends to use =, SameValue, is, or some wording like that. I think all three of those would mean the same thing here and each be acceptable.